lib/reel/websocket.rb

```diff
@@ -10,10 +9,11 @@ class WebSocket
 
     attr_reader :socket
     def_delegators :@socket, :addr, :peeraddr
+    def_delegators :@driver, :ping
 
     def initialize(info, connection)
-      driver_env = DriverEnvironment.new(info, connection.socket)      
-      
+      driver_env = DriverEnvironment.new(info, connection.socket)
+
       @socket = connection.hijack_socket
       @request_info = info
 
```

Comment:
Just under this point, can we not use `rack()` and use one of the other initialization methods?